hs-init.hs

```diff
@@ -188,11 +185,13 @@ generateProject repo owner description Targets{..} = do
   doScriptCommand =
     "chmod" ["+x", "b"]
 
-  doGithubCommands :: IO ()
-  doGithubCommands = do
+  doGithubCommands :: Bool -> IO ()
+  doGithubCommands private = do
     -- Create the repository on Github.
     "git" ["init"]
-    "hub" ["create", "-d", description, owner <> "/" <> repo]
+    -- creates private repository if asked so.
+    let priv :: Text = if private then "-p" else ""
+    "hub" ["create", priv, "-d", description, owner <> "/" <> repo]
```

Comment:
If you don't mind  writing `private` option at then end, you can write it like this:

```haskell
"hub" $ ["create", priv, "-d", description, owner <> "/" <> repo]
     ++ ["-p" | private]
```